data/commodities.txt

```diff
@@ -37,19 +68,30 @@ trade
 		"canned olives"
 		"canned soup"
 		"canned tuna"
+		"cantaloupes"
 		"cattle"
 		"cheese"
+		"cherries"
+		"chicken"
 		"chocolate"
+		"chewing gum"
 		"cloned meat"
+		"coconuts"
+		"coconut oil"
 		"coffee beans"
 		"cookies"
 		"cooking oils"
 		"corn oil"
 		"corn"
+		"courgette"
 		"crackers"
 		"croutons"
+		"cucumbers"
 		"cured meat"
 		"dairy products"
+		"dates"
+		"deer"
+		"deer meat"
```

Comment:
You already have `venison` and I don't think `deer meat` is a term in common usage, so I'd remove it.